test/specs/test.claim-compliance-feature-assurance.journeys.js

```diff
@@ -0,0 +1,92 @@
+import { browser, $, expect } from "@wdio/globals";
+import {
+  createSheepReviewClaim,
+  createSheepReviewForAdditionalHerd,
+  getDevSignInUrl,
+  enterPreMHReleaseDateAndContinue,
+} from "../utils/common.js";
+import { getClaimSelectorFromTable } from "../utils/backoffice-selectors.js";
+
+const assertAllClaimsAreInCheck = async (claimNumbers) => {
+  claimNumbers.forEach((claimNum) => expect(isClaimStatusInCheck(claimNum)).toBe(true));
+};
+
+const assertSomeClaimsAreOnHold = async (claimNumbers) => {
+  expect(claimNumbers.some((claimNum) => isClaimStatusOnHold(claimNum)).toBe(true));
+};
+
+const isClaimStatusInCheck = async (claimNumber) => {
+  const claimRow = $(getClaimSelectorFromTable(claimNumber)).parentElement();
+  return await claimRow.$('td[data-sort-value="IN CHECK"]').isDisplayed();
+};
+
+const isClaimStatusOnHold = async (claimNumber) => {
+  const claimRow = $(getClaimSelectorFromTable(claimNumber)).parentElement();
+  return await claimRow.$('td[data-sort-value="ON HOLD"]').isDisplayed();
+};
+
+describe("Test claim MH feature assurance compliance checks", async function () {
+  it("all claims go to in-check once a user has claimed for more than one herd of a given species", async () => {
```

Comment:
usually for the "it" syntax, the string is meant to follow on from the word "it", so here it should maybe say it("moves all claims to in-check once a user has claimed for more than one herd of a given species"